docs/notes.tex

```diff
@@ -400,19 +400,22 @@ \subsection{Types of coordinate systems used by basis functions}
 \begin{itemize}
 \item To treat all contractions to be spherical
   \begin{lstlisting}[xleftmargin=-25pt]
-output = high_level_function(basis, coord_type="spherical")
+basis = make_contractions(all_basis_dict, atoms, coords, coord_type="spherical")
 \end{lstlisting}
 \item To treat all contractions to be Cartesian
   \begin{lstlisting}[xleftmargin=-25pt]
-output = high_level_function(basis, coord_type="cartesian")
+basis = make_contractions(all_basis_dict, atoms, coords, coord_type="cartesian")
 \end{lstlisting}
 \item To treat first shell of generalized contractions to be Cartesian and
   second shell to be spherical
   \begin{lstlisting}[xleftmargin=-25pt]
-output = high_level_function(basis, coord_type=["cartesian", "spherical"])
+basis = make_contractions(
+    all_basis_dict, ["H", "H"], coords, coord_type=["cartesian", "spherical"]
+)
 \end{lstlisting}
-  In this case, the basis set must consist of exactly two shells of
-  generalized contractions. Otherwise, an error will raised.
+%  In this case, the basis set must consist of exactly two shells of
+%  generalized contractions. Otherwise, an error will be raised.
+In this case, each hydrogen atom must contributes exactly one shell of generalized contractions. Otherwise, an error will be raised.
```

Comment:
Maybe "contributes one list of generalized contractions"? As far as I know, what we call shells are contained in the GeneralizedContractionShell instances.

Reply:
I'm not even sure about this, just wondering.

Reply:
I was not sure about this either, but looking at the documentation, the term `shell of generalized contraction"  is defined is section 3.2 as a set of contractions with certain common properties (center, exponents and angular momentum), and then used through the document as such (e.g. section 4.4). So it is a list of generalized contractions like you point out, but with the specific characteristics mentioned. Also in make_contraction the keyword is used to create instances of GeneralizedContractionShell, so I think it is ok to refer to is a such. But let me know if it is more clear in another way.